.github/workflows/node_modules.yml

```diff
@@ -0,0 +1,258 @@
+name: npm install runner
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  start:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 5
+    permissions:
+      pull-requests: write
+    env:
+      GIT_DIR: git-dir.git
+      BASE_SHA: ${{ github.sha }}
+
+    steps:
+      - name: Interpret comment
+        id: interpret
+        uses: actions/github-script@v4
+        with:
+          result-encoding: string
+          script: |
+            // Make sure we're being spoken to
+            const { action, issue, comment: { body, id: comment_id } } = context.payload;
+            if (!(action === 'created' && body === '/npm install' && 'pull_request' in issue)) {
```

Comment:
Why does this have to be code? Couldn't the whole job be guarded by

    if: github.event.issue.pull_request && startsWith(github.event.comment.body, '/livetest')

? (action == created is already a top-level workflow condition). That would prevent spawning a VM for each and every PR comment,  also more declarative, and nicer because it's a proper skip instead of the code-induced "fail".

Reply:
For the reason stated in the comment: there's no real way to "exit" a workflow.  The best you can do is to fail it.  Otherwise you keep needing `if:` conditions on everything that follows.

To be honest, maybe that's worth removing the `/* HACK */`, saving the VM run, and seeing a bit less red on our actions page.  I'll take another look.

Reply:
If the first job gets skipped, all  subsequent jobs which `need:` it will get skipped too, no?

Reply:
Worth looking into, actually.

For steps, this is certainly not the case.  Skipping one step means the ones after it run still.  Even the word "skip" itself suggests going on to the next thing...

I'm happy to do it with conditions otherwise, anyway.

Reply:
Managed to get it working with explicit conditions on each job, which isn't so bad.